### PR TITLE
webtasks: task 2 added

### PR DESCRIPTION
**Pull request by gjstoychev**
Added button and function to fetch JSON data from test API

### CODE REVIEW

**Comment by mgenov** [comment]
PR is looking good to me, but I'm not able to merge it as branch is messed up with the master of upstream.

**Comment by mgenov** [comment]
Also I think that one of the task is already in master.

File: task2.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <title>Task 2</title>
+</head>
+
+<body>
+
+    <div><button type="button" onclick="ff()">Fetch JSON</button></div>
+    <div id="result">result goes here</div>
+
+    <script>
+        function ff() {
```
Comment (mgenov) [inline task2.html]: ff is not a good function name, fetchPosts sounds much better.

File: task2.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            var jsonTestUrl = 'https://jsonplaceholder.typicode.com/posts/1',
```
Comment (mgenov) [inline task2.html]: var postsUrl  = '...'

File: task2.html (hunk 1 above)
Comment (gjstoychev) [inline task2.html]: thank you. fixed

File: task2.html (hunk 2 above)
Comment (gjstoychev) [inline task2.html]: done

**Comment by mgenov** [comment]
There is still problem with merging of this PR.

**Pull request closed without merging**